lms/djangoapps/learner_dashboard/tests/test_programs.py

```diff
@@ -326,10 +328,23 @@ def setUp(self):
         self.set_program_in_catalog_cache(self.program_uuid, self.program)
         self.create_programs_config()
 
+        discussion_config = ProgramDiscussionsConfiguration.objects.create(
+            program_uuid=self.program_uuid,
+            enabled=True,
+            provider_type="piazza",
+        )
+        discussion_config.lti_configuration = LtiConfiguration.objects.create(
+            config_store=LtiConfiguration.CONFIG_ON_DB,
+            lti_1p1_launch_url='http://test.url',
+            lti_1p1_client_key='test_client_key',
+            lti_1p1_client_secret='test_client_secret',
+        )
+        discussion_config.save()
```

Comment:
I'd rather move this to a function, and call it in the tests which need this to be tested.